http/src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/SignedHttpClientConfigCallback.scala

```diff
@@ -52,7 +52,7 @@ class SignedHttpClientConfigCallback(
 ) extends HttpClientConfigCallback {
   private def clock(): LocalDateTime = LocalDateTime.now(ZoneId.of("UTC"))
   private val service = "es"
-  private val signer = AwsSigner(credentialsProvider, region, service, clock)
+  private val signer = AwsSigner(credentialsProvider, region, service, () => SignedHttpClientConfigCallback.this.clock())
```

Comment:
why was this necessary?

Reply:
to mute this sbt compile warning:
`[warn] snowplow-elasticsearch-loader/http/src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/SignedHttpClientConfigCallback.scala:55:72: Eta-expansion of zero-argument methods is deprecated. To avoid this warning, write (() => SignedHttpClientConfigCallback.this.clock()).
[warn]   private val signer = AwsSigner(credentialsProvider, region, service, clock)`